util/util.go

```diff
@@ -119,6 +119,20 @@ func TestEntPreCheck(t *testing.T) {
 	}
 }
 
+func TestRaftPreCheck(t *testing.T) {
```

Comment:
I'd like to switch the logic up a bit here by adding a `SKIP_RAFT_TESTS` variable instead. This means we only skip of the variable is set, and that the test will always be run otherwise.

It looks like you might be duplicating code that we are already have:

https://github.com/hashicorp/terraform-provider-vault/blob/66602ebb29019a52613818947a7b7f64b29d7dbe/util/util.go#L100-L107